flink-ml-iteration/src/main/java/org/apache/flink/iteration/proxy/state/ProxyOperatorStateBackend.java

```diff
@@ -58,12 +62,24 @@ public <K, V> BroadcastState<K, V> getBroadcastState(MapStateDescriptor<K, V> st
         return wrappedBackend.getBroadcastState(newDescriptor);
     }
 
+    private <S> TypeInformation<S> getTypeInfo(ListStateDescriptor<S> stateDescriptor)
+            throws Exception {
+        Field f = StateDescriptor.class.getDeclaredField("typeInfo");
```

Comment:
It is probably more readable to use `ReflectionUtils` in a way similar to `OutputReflectionContext`.